molr-mole-core/src/main/java/io/molr/mole/core/support/domain/VoidStub0.java

```diff
@@ -1,14 +1,29 @@
 package io.molr.mole.core.support.domain;
 
+import io.molr.commons.domain.Mission;
 import io.molr.commons.domain.Placeholder;
 
-public interface VoidStub0 extends MissionStub0<Void> {
+import static io.molr.commons.domain.Placeholders.returned;
 
-    <R> MissionStub0<R> returning(Class<R> returnType);
+public class VoidStub0 extends MissionStub0<Void> {
```

Comment:
When initiating a mission, I'm returning _OngoingMissionRun_ instead of _OngoingReturningMissionRun_ which is for Missions with an output value, thus the return type won't be of much use here. If we strictly don't want our class to have return type then we can make the _VoidStub_ classes a plain class instead of inheriting from _MissionStub_. What do you suggest? @kaifox